mock.rb

```diff
@@ -1,8 +1,25 @@
 require_relative './factorial_functions'
 
-describe Factorial do
-    it "finds the factorial of 5" do
-      calculator = Factorial.new
-      expect(calculator.factorial(5)).to eq(120)
-    end
-  end
\ No newline at end of file
+describe Solver do
+  it 'finds the factorial of 5' do
+    calculator = Solver.new
+    expect(calculator.factorial(5)).to eq(120)
+  end
+end
+
```

Comment:
- [ ] Good job with test coverage, but since we are aiming for 100% test coverage, it will be nice to add tests for a negative number, zero and one. Please kindly add tests for the factorial of a negative number, zero  and one.

Reply:
Thank you for your observation ! Indeed it is a good practice to test limit cases, so will do !